docs/docs/components/5-codecs/debezium.md

```
@@ -31,30 +28,37 @@ input:
     type: debezium_json
 ```
 
-## Output schema
+Deployment topology:
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)